docs/platforms/android/configuration/shared-environments.mdx

```diff
@@ -0,0 +1,60 @@
+---
+title: Using Sentry within an SDK
+description: "Learn how to use the Sentry SDK within a shared environment, e.g. another SDK."
+sidebar_order: 2000
+---
+
+<Note>
+
+Using the Sentry SDK within another SDK is [discouraged](https://docs.sentry.io/platforms/). This can lead to unexpected behaviour and potential data leakage. If you need to use Sentry within another SDK, please follow the best practices outlined below.
+
+</Note>
+
+<Note>
+
+When setting up Sentry in a shared environment where multiple Sentry instances may run, you should **not use `Sentry.init()`**, as this will pollute the global state.
+
+</Note>
+
```

Comment:
Are there any specific use cases like `Browser Extensions` in JS, just so readers won't confuse it with using RN and Android for example.

Reply:
Not really, but I rephrased it a bit to make it more clear - thanks!